src/Action.php

```diff
@@ -97,6 +119,35 @@ public function load($filename = null)
     }
 
     /**
+     * Loads a source file relatively from "actions/". If the filename is not
+     * provided it assumes the action name.
+     *
+     * @param callable $callback The callable to call.
+     * @return \Tao\Action
+     */
+    public function call(callable $callback = null)
+    {
+        if (!$callback) {
+            try {
+                $callback = require $this->getSourcePath(
```

Comment:
Not sure if you mean a log for any call being made, which should be logged in the first line, or a log just for when a callback is not provided, which I think should have another message.

I suppose you mean the first one, but please confirm.